addons/calendar/data/mail_template_data.xml

```diff
@@ -367,86 +353,84 @@
-                            </li>
-                        </t>
-                    </ul>
-                </td>
-            </tr>
-        </table>
-    </div>
-    <div class="user_input">
-        <hr/>
-        <p placeholder="Enter your message here"><br/></p>
-
+                    </t>
+                </span>
+            </div>
+            <div t-if="recurrent" style="margin-top: 5px;">
+                <span style="display: inline-block; width: 100px; opacity: 0.76;">When</span>
+                <t t-out="object.recurrence_id.get_recurrence_name() or ''">Every 1 Weeks, for 3 events</t>
+            </div>
+            <div t-if="not object.allday and object.duration" style="margin-top: 5px;">
+                <span style="display: inline-block; width: 100px; opacity: 0.76;">Duration</span>
+                <t t-out="('%dH%02d' % (object.duration,round(object.duration*60)%60))">0H30</t>
+            </div>
+            <div t-if="object.location" style="margin-top: 5px;">
+                <span style="display: inline-block; width: 100px; opacity: 0.76; vertical-align: top;">Location</span>
+                <div style="display: inline-block; max-width: 450px;">
+                    <t t-out="object.location or ''">Bruxelles</t>
+                    <a target="_blank" t-if="not is_online or is_online and object.location != object.appointment_type_id.location_id.name" t-attf-href="http://maps.google.com/maps?oi=map&amp;q={{object.location}}" style="margin-left: 12px; font-size: 12px; color: #008f8c; text-decoration: underline;">View Map</a>
+                </div>
+            </div>
+            <div t-if="object.videocall_location" style="margin-top: 5px;">
+                <span style="display: inline-block; width: 100px; opacity: 0.76;">
+                    <t t-if="object.videocall_source == 'discuss'">Join with</t>
+                    <t t-else="">Join</t>
+                </span>
+                <a t-attf-href="object.videocall_location" target="_blank" style="color: #008f8c;">
+                    <t t-if="object.videocall_source == 'discuss'">Odoo Discuss</t>
+                    <t t-else="">Video meeting</t>
+                </a>
+            </div>
+        </div>
+        <div style="margin: 32px 0 0;">
+            <h2 t-attf-style="margin-bottom: 12px; font-size: 14px; font-weight: bold; color: {{user.company_id.email_secondary_color or '#875A7B'}};">Attendees</h2>
+            <div t-foreach="object.attendee_ids" t-as="attendee" style="margin-bottom: 5px;">
+                <img t-if="attendee.state" t-attf-src="/calendar/static/src/img/state_{{ attendee.state }}.png" style="display: inline-block; border:0; outline:none; width: 15px; height: 15px; vertical-align: text-bottom;" />
+                <t t-if="attendee.common_name">
+                    <span style="margin-left: 6px" t-out="attendee.common_name or ''">Mitchell Admin</span>
+                </t>
+                <t t-else=""></t>
```

Comment:
I would say either empty either 'Attendee' and to align with templates in the other branch :) (I suggested 'Attendee' there)